android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -125,6 +125,12 @@ public SessionViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final SessionViewHolder holder, final int position) {
         Session session = getItem(position);
+
+        //removing session when in draft state
+        if(session.getState().equals("draft")) {
+            removeItem(session);
```

Comment:
nope. again not found.
can you please check and tell the problem because accessing  the list object over here doesnt seem possible to me.